src/commandHandlers/issue.ts

```diff
@@ -0,0 +1,60 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { Message, MessageEmbed } from 'discord.js';
+
+const req = async(q: string): Promise<any[]> => {
+    const config: AxiosRequestConfig = {
+        url: `https://api.github.com/search/repositories`,
+        params: { q }
+    };
+
+    if(process.env.GITHUB_TOKEN) config.headers = { 'Authorization': `token ${process.env.GITHUB_TOKEN}` };
```

Comment:
Could you add brackets to the if statement? To match project style.
example:
```typescript
if (x) {
  doStuff()
}
```

Reply:
I see there are few if statements like this

Reply:
Of course. That's usually my style on single line ifs and since the lint didn't complaint I though the style was present in this repo

Reply:
Hi Lucas 😃, you are right. Sorry but we are still working on adding Prettier and a linter, we have a PR for it. In the meantime, you don't have to worry too much about the styling issues. As a maintainer, we can commit suggestions on GitHub to match the style @AllanRegush said 👍.

Also, I can't mention you with @itsphaze. Are you on the EddieJaoudeCommunity GitHub organization? If not you can join [here](https://github.com/EddieJaoudeCommunity/support/issues/new/choose) by creating an issue with the invite template 😃.